### ISSUE TITLE
No issue just weird

### ISSUE THREAD

**Issue by DHStyle541** [Malcolm-Q/LC-SellableBodies#1]
Hey, add the Animation default game because if without animation so weird
and Drop location Thumper / Spider still False Location ( no fix )

i hope u listen this is :)

**Comment by Vej333** [comment]
Yeah, if there's a way to make it so that you can still see the dying animation, before they drop, or even better, if there was the option to choose afterward if we want to see the animation, it would be really cool.
Why the option to choose ? For comedic purposes. Imagine all enemies dies normally and you and your team play as if nothing's wrong, until you need a bit of extra cash for the quota before leaving.
The teamate who knows about the mod will just borrow a shotgun, kill a eyeless dog, who just drop instantly dead (no animation), in front of everyone, and in addition, get brought into the ship.
Then your teamate says, with a old timer's voice : "what a price we got'here. The hunt has been generous this season".

This would be perfect to get everyone confused in the ship.
